content/chainguard/chainguard-images/working-with-images/compiled-applications.md

```diff
@@ -0,0 +1,79 @@
+---
+title: "Choosing an Image for your Compiled Programs"
+linktitle: "Images for Compiled Programs"
+aliases: 
+type: "article"
+description: "An overview comparing various Chainguard Images for compiled programs"
+date: 2024-07-12T17:55:01+00:00
+lastmod: 2024-07-12T18:23:31+00:00
+draft: false
+tags: ["Chainguard Images", "Product"]
+images: []
+menu:
+  docs:
+    parent: "chainguard-images"
+weight: 200
+toc: true
+---
+
+When selecting the right base image for your application, there are a variety of factors to take into consideration. For starters, it is critical that your application has all of the dependencies it needs to run. The ideal base image will contain the essential packages you need, while leaving out the ones you don’t. However, in practice, you will need to build upon your images so they meet your specific needs, making it all the more important that you have a strong foundation.
+
+In this guide, we will explore a variety of Chainguard Images which are suitable for different compiled applications. We will take a look at their availability, packages, and use-case differences so you can move closer to settling on the best base image for your specific needs.
+
+## Available Images
+
+### static
+The Chainguard `static` base image is a Wolfi-based image available in one variant with the `:latest` tag. The `static` image is extremely minimal and is not intended to be run directly. It is used to host stand-alone, static binaries, like those produced by compilers such as `gcc`, `go`, and `rust`. It does not contain any programs you can run out-of-the-box. You must add your own static binaries to the image, for example using a Dockerfile multi-stage build.
+
+The following packages are included in the `static:latest` Chainguard Image:
```

Comment:
Would be curious to know what people think of adding the package listings in this cheatsheet. I felt it was appropriate considering users may wonder what each image includes to inform the differences between each, but wasn't sure if it was too cluttered.

Reply:
I think it's useful information, but I wonder if folks external to us would benefit much from the package names, as they can be a bit inscrutable. (Let me know if you think I'm wrong about this.) We could either add short descriptions of the packages or link out to some other description inwolfi.

I'd also say using a list format might help to make these more scannable and less cluttered looking, something like this (or with links if we go that route):

The following packages are included in the static:latest Chainguard Image: ca-certificates-bundle

- `chainguard-baselayout`
- `glibc-locale-posix`
- `tzdata`
- `wolfi-baselayout`